src/server.tsx

```diff
@@ -406,9 +439,55 @@ app.get(
     },
 );
 
+app.get(
+    '/amp/epic_view', // IMPORTANT: do not change this route!
+    cors({
+        origin: [
+            'https://amp-theguardian-com.cdn.ampproject.org',
+            'https://amp.theguardian.com',
+            'http://localhost:3030',
+            'https://amp.code.dev-theguardian.com',
+        ],
+        credentials: true,
+        allowedHeaders: ['x-gu-geoip-country-code'],
+    }),
+    async (req: express.Request, res: express.Response, next: express.NextFunction) => {
```

Comment:
I think again we can drop `next` here - in the case that it errors out, we could call `res.sendStatus(500)`

Reply:
I think we should keep these in for the time being, if only in the name of consistency across the server file. We could look at refactoring in a separate PR though :)